_spinnaker/terraform_alpha.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: post
+title: Terraform (alpha)
+order: 141
+---
+
+The following tutorial will walk you through how to setup the alpha version of our Terraform integration and execute Terraform code stored in a Github repo as part of a Spinnaker pipeline. We'll assume that you're using Terraform to create and manage infrastructure on AWS.
+
+
+## How to submit feedback
+
+If you decide to enable this feauture and have any feedback you'd like to submit, please let us know at [go.armory.io/ideas](go.armory.io/ideas)! We're constantly iterating on customer feedback to ensure that the features we build make your life easier!
+
+## Prerequisites
+
+1. A Armory Spinnaker installation running on Kubernetes and installed via Armory Halyard. ([instructions](/spinnaker/install)).
+2. A Github API token that has access to your Terraform project. See [this documentation](https://blog.github.com/2013-05-16-personal-api-tokens/) for details on how to generate this token.
+3. An AWS keypair with with enough permission to access your state store in S3 and manage the resources created by Terraform.
+
+## Installation
+
+#### Deploying the Terraform service
+
+The core of our Terraform integration is the Terraformer service. This is the service which is responsible for fetching your Terraform code and executing it. To do this, we'll need to install the following Kubernetes manifest using `kubectl`. *Note: we'll be adding support for deploying Terraformer via Halyard in future releases.* 
+
+You'll need to replace 4 references before doing so:
+1. `your-github-api-token` - the API token which will be used to fetch your repository
+2. `profile-name` - the name of the AWS profile which your Terraform code will use to provision infrastructure
+3. `your-aws-access-key` - the AWS Access Key ID of your keypair
+4. `your-aws-secret-access=key` - the AWS Secret Access Key of your keypair
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: terraformer-config
+  namespace: spinnaker
+data:
+  terraformer.yml: |
+    server:
+      host: 0.0.0.0
+      port: 7088
+    redis:
+      host: spin-redis
```

Comment:
Maybe we should clarify that Terraformer will use the internal Redis with this, some customers may have deleted it in favor of an external ElasticCache.

Reply:
good point. i'll just change it to use the SPEL from the default config.